dandi/cli/cmd_organize.py

```diff
@@ -212,6 +239,26 @@ def _get_metadata(path):
 
     metadata = create_unique_filenames_from_metadata(metadata)
 
+    # update metadata with external_file information:
+    external_files_missing_metadata_bool = [len(m["external_file_objects"]) == 0 for m in metadata]
+    if all(external_files_missing_metadata_bool) and rewrite == "external-file":
+        lgr.warning("rewrite option specified as 'external_file' but no external_files found"
+                    f"linked to any nwbfile found in {paths}")
```

Comment:
The proper way to format a log message is with `%s`-style specifiers and arguments, e.g., `lgr.warning("... linked to any nwbfile found in %s", paths)`.